bluepyopt/ephys/efeatures.py

```diff
@@ -22,6 +22,7 @@
 
 import logging
 
+from bluepyopt.ephys.serializer import DictMixin
```

Comment:
Up to you.  Are the other classes going to be serialized?  Won't they be more complicated, if they are?

I still think it might be worthwhile to break out the ephys portion, so that's another reason to leave it here.

I'm agnostic on the issue.